### DISCUSSION TITLE
Helpfiles

### DISCUSSION THREAD

**Discussion by Psypher9** [rezalas/riftshadow#219] [Q&A]
Would any of the in-game help documentation be a good fit for a GitHub wiki or is it better left in-game?

**Comment by rezalas** [comment]
I think that’s a good idea. Good documentation is always a good idea

Thanks,
Paul McDowell
________________________________
From: Turner Bass ***@***.***>
Sent: Friday, October 20, 2023 6:01:30 PM
To: rezalas/riftshadow ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [rezalas/riftshadow] Helpfiles (Discussion #219)


Would any of the in-game help documentation be a good fit for a GitHub wiki or is it better left in-game?
